app/src/main/res/layout/component_list_order_bar.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:background="@color/order_bar"
+    android:orientation="horizontal"
+    android:paddingStart="20dp"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@id/top_app_bar">
+
+    <TextView
+        android:id="@+id/list_order_type_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/white"
+        tools:text="Ordering By Name"/>
```

Comment:
Aqui ao invés de um tools pra string não seria um android:text já? A string também ficaria no arquivo de strings e não diretamente na view. O que tu havia pensado de diferente ?

Reply:
Tinha pensado em setar diretamente no código, junto com setar a ordenação inicial. Mas já dá para colocar como text mesmo

Reply:
Ah, agora entendi melhor. Achei que quando clicasse no change iria abrir uma modal para escolher por qual atributo ordenar e se seria ascendente ou descendente. Mas se for só pelo nome vou colocar direto no android:text mesmo

Reply:
corrigido